__tests__/ExpensiMark-HTML-test.js

```diff
@@ -664,3 +664,91 @@ test('Test for link with no content', () => {
     const resultString = '[  ](<a href="https://www.link.com" target="_blank" rel="noreferrer noopener">www.link.com</a>)';
     expect(parser.replace(testString)).toBe(resultString);
 });
+
+// Valid text that should match for user mentions
+test('Test for user mention with single username', () => {
+    const testString = '@username';
+    const resultString = '<mention-user>@username</mention-user>';
```

Comment:
As of right now, this isn't a valid mention. We only want to allow `@username@expensify.com` and `@+19728974297@expensify.sms` style mentions. We will add support for `@username` soon, but not quite yet. We shouldn't allow it yet because the back-end won't know what to do with it.

Reply:
>Should we disable it from the regex and add a comment how we can allow that in future incase@username becomes supported
I prefer this. Let's disable it from the regex for now.

Reply:
No. For now only @+19728974297@expensify.sms for phone numbers.